doc/source/dev/api-dev/array_api.rst

```diff
@@ -90,12 +90,19 @@ More strict array input validation will reject ``np.matrix`` and
 Currently supported functionality
 `````````````````````````````````
 
-The following modules provide Array API standard support when the environment
+The following modules provide array API standard support when the environment
```

Comment:
Sorry I didn't mention it before, but capitalization of "Array API" is inconsistent throughout this document. The consortium doesn't capitalize it; scikit learn seems to (and I tend to agree with them that it *should* be a proper noun).

Reply:
I guess we should stick with what the consortium does for now (making it consistent throughout this document) and open an issue if you think they should change to capitalisation.